llvm/test/CodeGen/SPIRV/inline/type.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-unknown-unknown %s -o - -filetype=obj | spirv-val %}
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-unknown-unknown %s -o - | spirv-as - -o - | spirv-val %}
+
+; CHECK: [[uint32_t:%[0-9]+]] = OpTypeInt 32 0
+
+; CHECK: [[image_t:%[0-9]+]] = OpTypeImage %3 2D 2 0 0 1 Unknown
+%type_2d_image = type target("spirv.Image", float, 1, 2, 0, 0, 1, 0)
+
+%literal_false = type target("spirv.Literal", 0)
+%literal_8 = type target("spirv.Literal", 8)
```

Comment:
These don't get used, and have no `CHECK`s for them. Could we add something where they are used to actually test them?

Reply:
These originated in the equivalent DXC test, which used them for `uint8_t`. I removed those parts since it would require the `Int8` capability, which I don't think I can add using LLVM IR right now – but I'll replace it with 32-bit signed integers instead.

Reply:
I fixed this by replacing the arbitrary integers with a new `OpUnknown` syntax. This syntax will be added to spirv-as by https://github.com/KhronosGroup/SPIRV-Tools/pull/6024.